.coderabbit.yml

```
@@ -44,6 +45,8 @@ reviews:
       instructions: |
         - 시간과 관련된 DB 스키마를 설정할 경우에는, TIMESTAMP가 아닌 DATETIME을 사용한다
         - Boolean Type을 사용할 경우, BIT를 사용해 통일성을 지킨다
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)